### ISSUE TITLE
preview-plus installing failed.

### ISSUE THREAD

**Issue by rawoon** [skandasoft/preview-plus#52]
npm WARN deprecated jade@1.11.0: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated react-tools@0.12.2: react-tools is deprecated. For more information, visit https://fb.me/react-tools-deprecated
npm ERR! Darwin 16.3.0
npm ERR! argv "/Users/rawoon89/Desktop/Atom.app/Contents/Resources/app/apm/bin/node" "/Users/rawoon89/Desktop/Atom.app/Contents/Resources/app/apm/node_modules/npm/bin/npm-cli.js" "--globalconfig" "/Users/rawoon89/.atom/.apm/.apmrc" "--userconfig" "/Users/rawoon89/.atom/.apmrc" "install" "/private/var/folders/nr/67ff5sx93kx85kgd4fyds8qw0000gn/T/d-1161123-2737-17ktfp7/package.tgz" "--runtime=electron" "--target=1.3.13" "--arch=x64" "--global-style"
npm ERR! node v4.4.5
npm ERR! npm  v3.10.5
npm ERR! code EREADFILE

npm ERR! Error extracting /Users/rawoon89/.atom/.apm/typescript-compiler/1.3.0/package.tgz archive: ENOENT: no such file or directory, open '/Users/rawoon89/.atom/.apm/typescript-compiler/1.3.0/package.tgz'
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /private/var/folders/nr/67ff5sx93kx85kgd4fyds8qw0000gn/T/apm-install-dir-1161123-2737-1h6u9ao/npm-debug.log

I want to see preview of html & css file in Atom and need this package.
But Its install failed with above error

**Comment by angwdev** [comment]
same problem

**Comment by agraubert** [comment]
I'm also having this problem.  Has anyone figured anything out?

**Comment by viceroyviki** [comment]
I had a similar error and finally got preview plus working. Here's what I did though I am not sure if this will work in your particular instance. But if it does update the same here and give me a thumbs up. If not update your issue, will see if I can troubleshoot further. Keep in mind, my system is Windows 10 Creator Build
1. Download the zip folder for preview-plus.
2. Navigate to the atom package folder, which in my case was in C:\Users\YOUR_COMPUTER_NAME\.atom\packages\
3. Unzip the preview-plus-master folder so a new package folder is created as such,
C:\Users\YOUR_COMPUTER_NAME\.atom\packages\preview-plus-master
4. Start Atom and install browser-plus if you haven't already done that.
5. Notice the errors thrown upon starting Atom and install them one by one to the folder 
C:\Users\YOUR_COMPUTER_NAME\.atom\packages\preview-plus-master
6. Please note that global installs does not work. You have to locally install each module to the preview-plus-master folder.
7. Each time you install a missing module, restart Atom until missing modules no longer exist.
8. Fire up the browser plus using Ctrl + Shift + P and typing browser plus. Click on live icon (looks like a thunderbolt)
9. Mark if the browser windows asks for any missing modules. In my case html2jade was missing.
10. Install all missing modules just as before locally to the preview-plus-master folder and keep restarting atom each time.
11. When all missing modules are installed, browser-plus and preview-plus should both work.

Basically, just unzip the master file from git for preview-plus in your Atom packages folder and keep installing each missing module locally in the preview-plus-master package folder until it works.
Hope this helps!

**Comment by gauravsitlani** [comment]
same problem, any progress?
i think it may be an npm issue.

**Comment by coviyarce** [comment]
Same problem, any suggestions?
